driver-examples/osgi/src/main/java/com/datastax/driver/osgi/impl/Activator.java

```diff
@@ -17,10 +17,13 @@
 
 import java.util.Hashtable;
 
+import com.sun.org.apache.xpath.internal.operations.Bool;
+import io.netty.channel.socket.SocketChannel;
```

Comment:
Wondering if there is a way we can test this in both the shaded and non-shaded case.  Maybe we can have 2 separate NettyCustomizer classes, one depending on the shaded SocketChannel and another the non-shaded version and then based on the System property we pull in a different customizer instance.   I think if the pom.xml for the osgi project depends on the shaded classifier and also pulls in netty we can do this with no compiler problems, and then pax-exam would load the proper one.  That may be a problem given that the exports for the Mailbox bundle will depend on netty explicitly in both cases.  Maybe we need to have separate bundles...I'll think on this some more and see if I can come up with a solution, but if you come up with one feel free to go ahead and implement it :)